modules/containers-signature-verify-enable.adoc

```diff
@@ -78,19 +79,90 @@ $ butane 51-worker-rh-registry-trust.bu -o 51-worker-rh-registry-trust.yaml
 $ oc apply -f 51-worker-rh-registry-trust.yaml
 ----
 
-. Create a Butane config and corresponding machine config which writes the configuration to disk on the control plane nodes:
+. Check that the worker machine config pool has rolled out with the new machine config:
+
+.. Check that the new machine config was created:
++
+[source,terminal]
+----
+$ oc get mc
+----
++
+.Sample output
+[source,terminal] 
+----
+NAME                                               GENERATEDBYCONTROLLER                      IGNITIONVERSION   AGE
+00-master                                          a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             25m
+00-worker                                          a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             25m
+01-master-container-runtime                        a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             25m
+01-master-kubelet                                  a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             25m
+01-worker-container-runtime                        a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             25m
+01-worker-kubelet                                  a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             25m
+51-master-rh-registry-trust                                                                   3.2.0             13s
+51-worker-rh-registry-trust                                                                   3.2.0             53s <1>
+99-master-generated-crio-seccomp-use-default                                                  3.2.0             25m
+99-master-generated-registries                     a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             25m
+99-master-ssh                                                                                 3.2.0             28m
+99-worker-generated-crio-seccomp-use-default                                                  3.2.0             25m
+99-worker-generated-registries                     a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             25m
+99-worker-ssh                                                                                 3.2.0             28m
+rendered-master-af1e7ff78da0a9c851bab4be2777773b   a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             8s  
+rendered-master-cd51fd0c47e91812bfef2765c52ec7e6   a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             24m
+rendered-worker-2b52f75684fbc711bd1652dd86fd0b82   a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             24m
+rendered-worker-be3b3bce4f4aa52a62902304bac9da3c   a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             48s <2>
+----
+<1> New machine config
+<2> New rendered machine config 
+
+.. Check that the machine config pool worker is updating with the new machine config:
```

Comment:
Change 'config' to 'configuration'. I am not sure of the context here, but looking at the single usage, it appears to need to the entire word.